src/lib/utils.ts

```diff
@@ -75,3 +75,30 @@ export function getAbsolutePath( filePath: string ): string {
 
 	return filePath;
 }
+
+/**
+ * Parse error object and return probable error.
+ *
+ * @param {Error} Error object
+ *
+ * @return {string|null} Error string when error was found, otherwise null.
+ */
+export function parseApiError( err: {
+	networkError?: { message?: string };
+	message?: string;
+	graphQLErrors?: { message?: string }[];
+} ): string | null {
+	if ( err?.networkError?.message ) {
+		return err?.networkError?.message;
+	}
+
+	if ( err?.graphQLErrors && err?.graphQLErrors?.length > 0 && err?.graphQLErrors[ 0 ]?.message ) {
+		return err?.graphQLErrors[ 0 ]?.message;
+	}
```

Comment:
[MINOR]
I'm pretty sure you only need this here:
```suggestion
	if ( err?.graphQLErrors?.[ 0 ]?.message ) {
		return err.graphQLErrors[ 0 ].message;
	}
```